### ISSUE TITLE
0 values is snown as No data

### ISSUE THREAD

**Issue by HenrikRolandHansen** [pierosavi/pierosavi-imageit-panel#83]
I'm using an influx database where the query returns 0 but the value is show as "No data"

![image](https://user-images.githubusercontent.com/8969659/110456209-91ede180-80c9-11eb-98d2-4ec3e048912d.png)

 - Browser edge
 - Plugin Version latest
 - Grafana Version [e.g. 7.4.3]

**Comment by pierosavi** [comment]
can you check that the plugin version is 1.0.2? this was fixed already

**Comment by HenrikRolandHansen** [comment]
Yes sorry it was not the latest version I had. After update it works

Sorry for the inconvenience
